CMakeLists.txt

```diff
@@ -1218,6 +1220,10 @@ add_function_test(perror
   SOURCES test/function/startup/perror.cpp
 )
 
+add_function_test(facility
```

Comment:
you'll need to change the name of this test from `facility` to something unique (probably `prival` is the best choice) in order for CMake to work properly.